int64 timestamp
string source_address
string anchor_address
string tag_address
int32 range_dist_mm
int32 range_rsl_mm
int32 range_raw_mm
float32 rsl
